### ISSUE TITLE
Binary Port Forwarding is using "-p" which is deprecated as for oc 3.11

### ISSUE THREAD

**Issue by adietish** [openshift/openshift-restclient-java#339]
OpenShiftBinaryPortForwarding is using "-p" to provide the pod name when port-forwarding.
```
static class PodName implements OpenShiftBinaryOption {

        private IPod pod;

        public PodName(IPod pod) {
            this.pod = pod;
        }

        @Override
        public void append(StringBuilder commandLine) {
            commandLine.append(" -p ").append(pod.getName());
        }
    }
```
This option was deprecated as for oc 3.11:
```
Options:
-p, --pod='': Pod name (deprecated)
```
We should change our code to provide the pod name in the updated fashion (without '-p')

for further details see https://issues.jboss.org/browse/JBIDE-26251

**Comment by adietish** [comment]
for further details see https://issues.jboss.org/browse/JBIDE-26251

**Comment by adietish** [comment]
[test]

**Comment by adietish** [comment]
resolved by the fix in #340
